docs/messaging/messaging-spans.md

```diff
@@ -216,20 +216,14 @@ Span kind SHOULD be set according to the following table, based on the operation
 | Operation type | Span kind|
 |----------------|-------------|
 | `create`       | `PRODUCER` |
-| `publish`      | `PRODUCER` if the context of the "Publish" span is used as creation context. |
+| `publish`      | `PRODUCER` if the context of the "Publish" span is used as creation context, otherwise `CLIENT`. |
 | `receive`      | `CONSUMER` |
-| `process`      | `CONSUMER` for push-based scenarios where no "Receive" span exists. |
```

Comment:
Based on the discussion in the Messaging SIG 6/20 we should consider `Internal` kind

Reply:
since `recieve` is `client` now, we can always use `consumer` for the `process` span